packages/fast-animation/package.json

```diff
@@ -87,11 +88,16 @@
     "tslint-loader": "^3.5.3",
     "tslint-react": "^3.0.0",
     "typescript": "^3.0.1",
-    "webpack": "^2.6.0",
+    "webpack": "^4.28.4",
     "webpack-bundle-analyzer": "^2.8.2",
-    "webpack-dev-server": "^2.4.5"
+    "webpack-cli": "^3.2.1",
+    "webpack-dev-server": "^3.1.14"
   },
   "peerDependencies": {
-    "lodash-es": "^4.0.0"
+    "ajv": "^6.0.0",
+    "babel-core": "6.0.0",
+    "lodash-es": "^4.0.0",
+    "typescript": "^2.2.0",
+    "webpack": "^2.2.0"
```

Comment:
webpack, babel, typescript and ajv are not needed from the export of this package, why are they peer dependencies?

Reply:
I was getting console warnings that these peer dependencies were required.

Reply:
I will remove these peer dependencies.